packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageSettingsModal.tsx

```diff
@@ -1,140 +1,83 @@
-import { useCallback, useContext, useEffect } from 'react'
+import { useCallback, useEffect, useState } from 'react'
 
 import {
-  useApproveManagedAccount,
-  useGetManagedAccounts,
-  useRejectManagedAccount
-} from '@audius/common/api'
-import { Status, UserMetadata } from '@audius/common/models'
-import { accountSelectors } from '@audius/common/store'
-import {
-  Box,
-  Divider,
-  Flex,
   IconUserArrowRotate,
   Modal,
-  ModalContent,
+  ModalContentPages,
   ModalHeader,
   ModalProps,
-  ModalTitle,
-  Text,
-  TextLink
+  ModalTitle
 } from '@audius/harmony'
 
-import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
-import { ToastContext } from 'components/toast/ToastContext'
-import { useSelector } from 'utils/reducer'
-
-import { AccountListItem } from './AccountListItem'
-import { sharedMessages } from './sharedMessages'
-const { getUserId } = accountSelectors
+import { AccountsYouManageHomePage } from './AccountsYouManageHomePage'
+import { StopManagingConfirmationPage } from './StopManagingConfirmationPage'
+import { AccountsYouManagePages, AccountsYouManagePagesParams } from './types'
 
 const messages = {
   accountsYouManage: 'Accounts You Manage',
-  takeControl:
-    'Take control of your managed accounts by making changes to their profiles, preferences, and content.',
-  noAccounts: 'You don’t manage any accounts.',
-  somethingWentWrong: 'Something went wrong. Please try again later.'
+  stopManaging: 'Stop Managing?'
 }
 
-type AccountsManagingYouSettingsModalProps = Omit<ModalProps, 'children'>
+const PAGE_TO_TITLE = {
+  [AccountsYouManagePages.HOME]: messages.accountsYouManage,
+  [AccountsYouManagePages.STOP_MANAGING]: messages.stopManaging
+}
+
+type AccountsYouManageSettingsModalProps = Omit<ModalProps, 'children'>
+
+const getCurrentPage = (currentPage: AccountsYouManagePages) => {
+  switch (currentPage) {
+    case AccountsYouManagePages.HOME:
+      return 0
+    case AccountsYouManagePages.STOP_MANAGING:
+      return 1
+  }
+}
 
 export const AccountsYouManageSettingsModal = (
-  props: AccountsManagingYouSettingsModalProps
+  props: AccountsYouManageSettingsModalProps
 ) => {
-  const userId = useSelector(getUserId) as number
-  const { data: managedAccounts, status } = useGetManagedAccounts({ userId })
-  const [approveManagedAccount, approveResult] = useApproveManagedAccount()
-  const [rejectManagedAccount, rejectResult] = useRejectManagedAccount()
-  const { toast } = useContext(ToastContext)
+  const { isOpen } = props
+  const [currentPage, setCurrentPage] = useState(AccountsYouManagePages.HOME)
 
-  const handleApprove = useCallback(
-    ({
-      currentUserId,
-      grantorUser
-    }: {
-      currentUserId: number
-      grantorUser: UserMetadata
-    }) => {
-      approveManagedAccount({ userId: currentUserId, grantorUser })
-    },
-    [approveManagedAccount]
-  )
+  const [currentPageParams, setCurrentPageParams] = useState<
+    AccountsYouManagePagesParams | undefined
```

Comment:
I could be wrong, but doesn't `useState` default to an optional value if you don't pass anything in the constructor? i.e. don't need to do `| undefined`

Reply:
I think you're right, will clean up the undefined